### PR TITLE
Migrate to .NET Standard 1.1

### PR DESCRIPTION
**Pull request by tilutza**
I have migrated the Firebase.Auth project to .net standard. However, I am not pretty sure how we can proceed with tests. 

Do you have any idea ?

### CODE REVIEW

File: src/Firebase.Auth/project.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿{
+  "supports": {},
+  "dependencies": {
+    "Microsoft.NETCore.Portable.Compatibility": "1.0.1",
+    "NETStandard.Library": "1.6.0",
```
Comment (bezysoftware) [inline src/Firebase.Auth/project.json]: This version is too high to make it usable in UWP. Can it target 1.4?

**Comment by tilutza** [comment]
As per documentation here, a lower version would target more platforms, thus i changed to 1.1

https://github.com/dotnet/standard/blob/master/docs/versions.md

File: src/Firebase.Auth.Tests/IntegrationTests.cs [hunk 2]
```diff
@@ -154,7 +154,7 @@ public void GetLinkedAccountsTest()
             var authProvider = new FirebaseAuthProvider(new FirebaseConfig(ApiKey));
             var email = $"abcd{new Random().Next()}@test.com";
 
-            var auth = authProvider.CreateUserWithEmailAndPasswordAsync(email, "test1234").Result;
+            var auth = authProvider.CreateUserWithEmailAndPasswordAsync(email, "test1234", false).Result;
```
Comment (lnicola) [inline src/Firebase.Auth.Tests/IntegrationTests.cs]: Why was this needed?

File: src/Firebase.Auth/project.json (hunk 1 above)
Comment (tilutza) [inline src/Firebase.Auth/project.json]: The target framework is 1.1. The standard library is for compatibility purposes between the old runtime and the 1.6
